hermes-mock/src/test/groovy/pl/allegro/tech/hermes/mock/HermesMockAvroTest.groovy

```diff
@@ -288,6 +288,33 @@ class HermesMockAvroTest extends Specification {
             1 == hermes.query().countMatchingAvroMessages(topicName, schema, TestMessage, filter)
     }
 
+    def "should reset received requests with Avro messages"() {
+        given:
+        def topicName = "my-test-avro-topic"
+        hermes.define().avroTopic(topicName)
+        publish(topicName, new TestMessage("test-key", "test-value"))
+
```

Comment:
I know it is already tested in other scenario but maybe we can make an assertion in the `given` section just for readability:
`hermes.query().countAvroMessages(topicName) == 1`

Reply:
Good idea. I've added this assertion.